core/templates/dev/head/exploration_editor/ExplorationEditor.js

```diff
@@ -923,17 +923,7 @@ oppia.controller('ExplorationSaveAndPublishButtons', [
 
         modalInstance.result.then(function(commitMessage) {
           _modalIsOpen = false;
-          saveDraftToBackend(commitMessage, function() {
-            explorationData.getLastSavedData().then(function(data) {
-              explorationData.getData().then(function(currentData) {
-                if (data.version > currentData.version) {
-                  autosaveInfoModalsService.showVersionMismatchModal(
-                    changeListService.getChangeList());
-                  return;
-                }
-              });
-            });
-          });
+          saveDraftToBackend(commitMessage);
```

Comment:
I've been comparing this to the autosaveChangeListOnChange function and that seems to behave a bit differently; it looks at the response it gets and uses this to determine whether to open the version-mismatch modal (rather than using explorationData.getData() to make an additional server call and check the versions that way. Is it possible to do the same thing in this case? Otherwise I'm concerned that the version-mismatch modal will not be opened and just a generic error message will be shown instead.